### PR TITLE
Update.deps.ziti edge 138

### PR DESCRIPTION
**Pull request by andrewpmartinez**

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -35,6 +40,77 @@ If paired with a ziti controller/routers which support terminator strategies for
...
+
+# Release 0.13.5
+## Theme
+
+  * Adds ability to verify 3rd party CAs via the CLI in the Ziti Edge API
+
+## Ziti CLI Verify CA Support
+
+Previous to this version the CLI was only capable of creating, editing,
+and deleting CAs. Verification of a CA to make it useful for auto CA
```
Comment (dovholuknf) [inline doc/release-notes.md:56]: this is wordy imo:
"Verification of a CA to make it useful for auto CA
enrollment and authentication required manually created HTTP requests
and `openssl` signing commands to create a verification cert."

maybe:
The act of verifying newly created CAs was cumbersome and required manually created HTTP requests
and using `openssl` signing commands to create a verification cert

Comment (andrewpmartinez) [reply doc/release-notes.md]: It sure was. I was already fixing it.

**Comment by andrewpmartinez** [comment]
Smoke tests passed. Merging

**Comment by andrewpmartinez** [comment]
https://jenkinstest.tools.netfoundry.io/blue/organizations/jenkins/ziti-smoke-test/detail/ziti-smoke-test/2682/pipeline/

**Pull request merged**